tools/ansible/tasks/wsu/main.yaml

```diff
@@ -0,0 +1,54 @@
+---
+- hosts: localhost
+  vars:
+    project_root: "{{ playbook_dir }}/../../../.."
+    kubelet_location: "https://dl.k8s.io/v1.14.0/kubernetes-node-windows-amd64.tar.gz"
+    tmp_path: "/tmp/wsu"
+
+  tasks:
+    - name: Gather required files
+      block:
+
+        - name: Create directory to store files to be used later
+          file:
+            path: "{{ tmp_path }}"
+            state: directory
+
+        - name: Build WMCB
+          make:
+            target: build
+            chdir: "{{ project_root }}"
+        - name: Move WMCB to temporary directory
+          command: mv "{{ project_root }}/wmcb.exe" "{{ tmp_path }}/wmcb.exe"
+
+        - name: Download Windows node kubelet
+          get_url:
+            url: "{{ kubelet_location }}"
+            dest: "{{ tmp_path }}/kube.tar.gz"
+
+        - name: Extract kubelet
+          unarchive:
+            src: "{{ tmp_path }}/kube.tar.gz"
+            dest: "{{ tmp_path }}"
+
+        - name: Grab kubelet from extracted directories
+          copy:
+            src: "./tmp/kubernetes/node/bin/kubelet.exe"
+            dest: "./tmp/kubelet.exe"
+
+- hosts: win
+  vars:
+    tmp_path: "{{ playbook_dir }}/tmp"
+
+  tasks:
+    - name:  Copy required files to Windows host
+      win_copy:
+        src: "{{ tmp_path }}/"
+        dest: "C:\\Windows\\Temp"
```

Comment:
Windows has a win_tempfile module too. And I think we should delete the tempfile created on Windows host after the task is done

Reply:
I think there is some merit in having files be in a consistent spot, for predictability and debugging purposes, since we are interacting with these files with wmcb.exe.
I think we can change this in the future if we don't find the consistency useful though

Reply:
I prefer that we use the `win_tempfile` module if you are going the temp directory route. If you want a consistent spot, then create one outside of the `temp` directory structure. Otherwise what guarantee do we have for the files to be present there as the admin could clear the global temp directory.

Reply:
Changed